index.js

```diff
@@ -113,7 +113,11 @@ exports.fromMapFileComment = function (comment, dir) {
 
 // Finds last sourcemap comment in file or returns null if none was found
 exports.fromSource = function (content, largeSource) {
-  if (largeSource) return convertFromLargeSource(content);
+  if (largeSource) {
+    var res = convertFromLargeSource(content);
+    if (typeof res === 'undefined') res = null;
+    return res;
```

Comment:
The `undefined` is fine but first reassigning `res` when all you do is return right after is a bit too complex.
Just turn it into a one liner. In this case it is also easier to just check for falsiness since you wanna return with `null` either way.